### PR TITLE
ZPI-xx: Small fixes

### PR DESCRIPTION
**Pull request by samuelus**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ on:
     branches: [ "main" ]
     paths:
       - 'charts/kam/Chart.yaml'
+  pull_request:
+    branches: [ "main" ]
```
Comment (Ninja00Shadow) [inline .github/workflows/docker.yml:9]: Isn't this too much?

File: frontend/env.sh [hunk 2]
```diff
@@ -11,5 +11,8 @@ do
     key=$(echo $i | cut -d '=' -f 1)
     value=$(echo $i | cut -d '=' -f 2-)
     echo "Replacing ${key} with ${value}"
-    find /usr/share/nginx/html -type f \( -name '*.js' -o -name '*.css' \) -exec sed -i "s|${key}|${value}|g" '{}' +
+
+    # Szukanie i podmiana z wyświetleniem linii
```
Comment (Ninja00Shadow) [inline frontend/env.sh:15]: Chat?

**Comment by samuelus** [comment]
image

**Review by Ninja00Shadow — APPROVED** [review]
I don't know what happened.

something wrong i can feel it

**Pull request merged**